@sentry/node memory leaks on node 10.13.0

## Package + Version

- [x] `@sentry/browser`
- [x] `@sentry/node`

### Version:

```
4.3.4
```

## Description

I tried integrating sentry into ah next.js project. I tried it using this template https://github.com/sheerun/next.js/tree/with-sentry-fix/examples/with-sentry and discovered what seems to be a memory-leak in sentry. If you check out this project and add memwatch

```
if (!dev) {
  memwatch.on("leak", info => {
    console.log("memwatch::leak");
    console.error(info);
  });

  memwatch.on("stats", stats => {
    console.log("memwatch::stats");
    console.error(Util.inspect(stats, true, null));
  });
}
```
and bombard the server with requests, i requested the following ressources: 

```
    "/",
    "/_next/static/r1zovjaZ1TujaA0hJEp91/pages/_app.js",
    "/_next/static/r1zovjaZ1TujaA0hJEp91/pages/_error.js",
    "/_next/static/r1zovjaZ1TujaA0hJEp91/pages/index.js",
    "/_next/static/runtime/main-1eaa6d1d0c8e7d048efd.js",
    "/_next/static/chunks/commons.b34d260fee0c4a698139.js",
    "/_next/static/runtime/webpack-42652fa8b82c329c0559.js"
```

With this the memory-usage grows endlessly for me. As soon as i remove the request and errorHandler from server.js the memory-leak stops. So it seems to be connected to those 2

Using [lab](https://github.com/hapijs/lab) for unit testing. The leaks are still present. I know leaks can be painful to debug. Is there an ETA for a fix?

> 1 tests complete
> Test duration: 1832 ms
> The following leaks were detected:__extends, __assign, __rest, __decorate, __param, __metadata, __awaiter, __generator, __exportStar, __values, __read, __spread, __await, __asyncGenerator, __asyncDelegator, __asyncValues, __makeTemplateObject, __importStar, __importDefault
> 
> npm ERR! code ELIFECYCLE
> npm ERR! errno 1
> npm ERR! theregulars-reviews-backend@0.0.1 test: `lab build/test`
> npm ERR! Exit status 1
> npm ERR! 
> npm ERR! Failed at the theregulars-reviews-backend@0.0.1 test script.
> npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
> 
> npm ERR! A complete log of this run can be found in:
> npm ERR!     /Users/sunknudsen/.npm/_logs/2019-02-13T14_41_28_595Z-debug.log

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -3,6 +3,7 @@
 ## 5.4.1
 
 - [integrations] fix: Tracing integration fetch headers bug.
+- [node] fix: Force agent-base to be at version 4.3.0 to fix various issues. Fix #1762, fix #2085
 
 ## 5.4.0
 

--- a/package.json
+++ b/package.json
@@ -56,5 +56,8 @@
     "typedoc": "^0.14.2",
     "typescript": "^3.4.5",
     "typescript-tslint-plugin": "^0.3.1"
+  },
+  "resolutions": {
+    "**/agent-base": "4.3.0"
   }
 }

